src/OpenApi/src/Services/OpenApiOptions.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.OpenApi;
+
+namespace Microsoft.AspNetCore.OpenApi;
+
+/// <summary>
+/// Options to support the construction of OpenAPI documents.
+/// </summary>
+public class OpenApiOptions
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="OpenApiOptions"/> class
+    /// with the default <see cref="ShouldInclude"/> predicate.
+    /// </summary>
+    public OpenApiOptions()
+    {
+        ShouldInclude = (description) => description.GroupName == null || description.GroupName == DocumentName;
```

Comment:
Why isn't this an initializer, like the other properties?

Reply:
The value we're using for the initialization references another property which is [a no-go](https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0236).